platforms/jvm/code-quality/src/testFixtures/groovy/org/gradle/quality/integtest/fixtures/PmdCoverage.groovy

```diff
@@ -20,7 +20,7 @@ package org.gradle.quality.integtest.fixtures
 import org.gradle.api.plugins.quality.PmdPlugin
 
 class PmdCoverage {
-    private final static List<String> ALL = [PmdPlugin.DEFAULT_PMD_VERSION, '4.3', '5.0.5', '5.1.1', '5.3.3', '6.0.0' /*Java 9*/, '6.13.0' /*Java 12*/, '7.0.0'].asImmutable()
+    private final static List<String> ALL = [PmdPlugin.DEFAULT_PMD_VERSION, '5.0.5', '5.1.1', '5.3.3', '6.0.0' /*Java 9*/, '6.13.0' /*Java 12*/, '7.0.0'].asImmutable()
```

Comment:
```suggestion
    private final static TESTED_PMD_VERSIONS  = [
        (JavaVersion.VERSION_1_8): [PmdPlugin.DEFAULT_PMD_VERSION, '5.0.5', '5.1.1', '5.3.3', '6.0.0'],
        (JavaVersion.VERSION_1_9): ['6.13.0'],
        (JavaVersion.VERSION_12): ['7.0.0']
    ]
```

💭 Something like this might be instead of the weird embedded comments in the middle of the list.  Then the getter would become:

```groovy
    static getSupportedVersionsByJdk() {
        TESTED_PMD_VERSIONS.values().flatten()
    }
```

Reply:
I think I can remove those comments and just redirect to 
https://github.com/gradle/gradle/blob/3deb4d83ba62d4490911a74784703ced9c725175/platforms/jvm/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/pmd/AbstractPmdPluginVersionIntegrationTest.groovy#L33-L56
WDYT?

Reply:
Even better!